test/handshake_helper.c

```diff
@@ -374,6 +374,22 @@ static void configure_handshake_ctx(SSL_CTX *server_ctx, SSL_CTX *server2_ctx,
     OPENSSL_assert(SSL_CTX_set_tlsext_ticket_keys(server_ctx, ticket_keys,
                                                   ticket_key_len) == 1);
     OPENSSL_free(ticket_keys);
+
+#ifndef OPENSSL_NO_CT
+    OPENSSL_assert(SSL_CTX_set_default_ctlog_list_file(client_ctx));
```

Comment:
This has no business being an assertion.  Assertions are for bugs in our code, or sometimes invalid caller arguments, never for "unexpected", but possible conditions, such as not being able to open some file, ...